### DISCUSSION TITLE
option to force players into spectator mode when they use /tp to teleport to a player

### DISCUSSION THREAD

**Discussion by felix920506** [EssentialsX/Essentials#4814] [Ideas and Feature Suggestions] [answered]
An option to automatically switch players into spectator mode when they use /tp to teleport would be great. it would not disturb whomever is the target.
How I would like it to work:
`/tp x y z`, `tp player x y z` will stay the same while `/tp player` will switch the person who issued the command to spectator mode when used

**Comment by Chew** [comment, marked as answer]
Whilst a good suggestion, this seems like something better suited for a plugin using the Essentials (or, hell, even just the Bukkit) API.
